src/main/java/supersymmetry/api/recipes/SuSyRecipeMaps.java

```diff
@@ -108,7 +110,7 @@ public class SuSyRecipeMaps {
             .setSlotOverlay(true, false, SusyGuiTextures.LARGE_REACTOR_ITEM_OVERLAY)
             .setSound(GTSoundEvents.CHEMICAL_REACTOR);
 
-    public static final RecipeMap<CatalystRecipeBuilder> POLYMERIZATION_RECIPES = new RecipeMap<>("polymerization_tank", 2, 1, 3, 1, new CatalystRecipeBuilder(), false)
+    public static final RecipeMap<CatalystRecipeBuilder> POLYMERIZATION_RECIPES = new RecipeMap<>("polymerization_tank", 3, 1, 3, 2, new CatalystRecipeBuilder(), false)
```

Comment:
what did you base this branch on?

Reply:
i based it off main then cherry picked the commit to a new branch and then realized I made a mistake and tried going back and undoing all the new changes